AnnotationInject.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.author                  = 'pjechris'
 
   s.ios.deployment_target   = "8.0"
-  s.swift_version           = "4.2"
+  s.swift_version           = ['4.2', '5.0','5.1', '5.2', '5.3', '5.4' ,'5.5', '5.6', '5.7']
```

Comment:
@david-spaeth Based on doc I think you need to add a "s" to `swift_version`
https://guides.cocoapods.org/syntax/podspec.html#swift_versions